src/api/init.ts

```diff
@@ -120,7 +120,7 @@ export async function startApiServer(opts: {
                 break;
               }
             }
-            return pathTemplate;
+            return '<invalid_path>';
```

Comment:
I might be misunderstanding something, but won't this always return `<invalid_path>`, even if `pathTemplate` is successfully set to a valid `pathRegex.path` in the for loop's condition above?

Reply:
Good catch, fixed!